### PR TITLE
Sean/no lowcounts in winchart

### PR DESCRIPTION
**Pull request by seangeggie**
Leave out players with <25% of played matches from stats and winchart.

### CODE REVIEW

**Review by netromdk — CHANGES_REQUESTED** [review]
These are great changes!

I only had one request.

File: tourney/stats.py [hunk 1]
```diff
@@ -335,10 +335,14 @@ def __fmt_num(self, num):
   def __fmt_top(self, lst, top_range, lookup):
     """Expects that `self.__personal` has already been filled!"""
     res = ""
+
+    req_plays = max([p["total_matches"] for p in self.__personal.values()]) / 4
+    qualifying_lst = [p for p in lst if self.__personal[p[0]]["total_matches"] > req_plays]
```
Comment (netromdk) [inline tourney/stats.py:340]: Having the exact number of required plays should still be accepted:
```suggestion
    qualifying_lst = [p for p in lst if self.__personal[p[0]]["total_matches"] >= req_plays]
```

Comment (seangeggie) [reply tourney/stats.py:340]: Oh yeah, that seems fair

**Review by netromdk — APPROVED** [review]
Thanks!

**Pull request merged**